packages/eslint-plugin-choreography-ts/tests/no-outside-choreographic-operator.test.ts

```diff
@@ -0,0 +1,318 @@
+                "seller",
+                (unwrap) => deliveryDateTable.get(unwrap(titleAtSeller))
+              );
+            }
+          }
+        )
+      }`,
+      errors: [
+        {
+          messageId: "error",
+          suggestions: [
+            {
+              messageId: "suggestion",
+              output: `
+      const test2: Choreography<Locations> = async ({
+        locally,
+        broadcast,
+        colocally,
+      }) => {
+        const [deliveryDateAtBuyer] = await colocally(
+          ["buyer1", "seller"],
+          async ({ peel, locally }) => {
+            const sharedDecision = peel(decision);
+            if (sharedDecision) {
+              const deliveryDateAtSeller = await locally(
+                "seller",
+                (unwrap) => deliveryDateTable.get(unwrap(titleAtSeller))
+              );
+            }
+          }
+        )
+      }`,
+            },
+          ],
+        },
+      ],
+    },
+    {
+      name: "test for proper insertion of missing nested operator into empty dependencies parameter",
+      code: `
+      const test2: Choreography<Locations> = async ({
+        locally,
+        broadcast,
+        colocally,
+      }) => {
+        const [deliveryDateAtBuyer] = await colocally(
+          ["buyer1", "seller"],
+          async ({}) => {
+            const sharedDecision = peel(decision);
+          }
+        )
+      }`,
+      output: `
+      const test2: Choreography<Locations> = async ({
+        locally,
+        broadcast,
+        colocally,
+      }) => {
+        const [deliveryDateAtBuyer] = await colocally(
+          ["buyer1", "seller"],
+          async ({ peel }) => {
+            const sharedDecision = peel(decision);
+          }
+        )
+      }`,
+      errors: [
+        {
+          messageId: "error",
+        },
+      ],
+    },
+    {
+      name: "test for insertion of missing nested operator in correct nested context",
+      code: `
+      type Locations = "alice" | "bob" | "carol";
+      const _test2: Choreography<Locations> = async ({ colocally }) => {
+        await colocally(
+          ["alice", "bob", "carol"],
+          async ({ colocally }) => {
+            await colocally(
+              ["alice", "bob"],
+              // ↓ should contain 'locally'
+              async ({}) => {
+                await locally("alice", () => {
+                  console.log("Hi, I'm Alice!");
+                });
+                return [];
+              },
+              []
+            );
+            return [];
+          },
+          []
+        );
+        return [];
+      };`,
+      output: `
+      type Locations = "alice" | "bob" | "carol";
+      const _test2: Choreography<Locations> = async ({ colocally }) => {
+        await colocally(
+          ["alice", "bob", "carol"],
+          async ({ colocally }) => {
+            await colocally(
+              ["alice", "bob"],
+              // ↓ should contain 'locally'
```

Comment:
```suggestion
```

Reply:
Oh ok, I see what you mean. That makes sense since it's repeated in all the test files. I can do that